### PR TITLE
docs(datasets and tables): replacing current table-level documentation with links to dbt-docs

### LINKED ISSUE #1499: docs: pointing existing `Datasets & Tables` docs to new dbt-docs site
**Is your feature request related to a problem? Please describe.**
As we've implemented the new dbt architecture and generated the new dbt-docs site, we want to replace the existing and now defunct content by pointing the docs at the dbt site. 

THIS ISSUE INCLUDES IDENTIFYING WHICH DOCS SHOULD NOT YET BE DEPRECATED - INCLUDING AIRTABLE, TRANSIT DATABASE, ANYTHING NOT YET IN DBT

**Describe the solution you'd like**
Replace the existing content and create new content that lists what is available at the dbt-docs site and provides links to the site.

**Describe alternatives you've considered**
We could potentially leave the docs as they are but they are already difficult to maintain and outdated. Using the dbt docs as our reference provides an accurate source of truth that is close to the code itself.

**Additional context**

### PR DESCRIPTION
**Pull request by charlie-costanzo**
# Description

As part of deploying dbt we benefit from new, automatically generated, table-level documentation that is easier to maintain, lives close to the code, and serves as our source of truth.

To make this useful, we need to remove the current documentation (keeping what doesn't yet live in dbt) and link out to the dbt site.

The dbt-docs that will be referenced are [linked here](https://dbt-docs.calitp.org/#!/source/source.calitp_warehouse.gtfs_rt_external_tables.service_alerts)

It is important to note that **there are some docs that should not yet be deprecated** including airtable, transit database, and anything not yet in dbt.

Datasets included in this PR:
* external_gtfs_rt - not yet added
* gtfs_schedule
* gtfs_schedule_history
* gtfs_schedule_type2
* views

to include?
* gtfs_rt?
* gtfs_rt_logs?
* gtfs_views_staging?
* information_schema?
* staging?

Resolves #1499 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation
- [ ] agencies.yml

### CODE REVIEW

**Review by lauriemerrell — COMMENTED** [review]
I have an overall question here -- do we need to maintain the full list of tables in both locations...? Would it be possible to:

1. Do any kind of programmatic embedding of table lists pulled directly from the dbt docs
2. Simply add docs to docs.calitp.org explaining how to navigate the dbt docs and, for each domain area, specifying what schema (dataset) in the warehouse contains the relevant data, but not listing out all the tables manually?

I am mostly just wondering about keeping this up to date.... I appreciate all the work done here to link everything, but I thought part of the value of the dbt docs is that ideally we don't have to manually re-generate docs (for example if a table name changes or a new table is added).

**Review by atvaccaro — CHANGES_REQUESTED** [review]
Yeah I think I agree with Laurie. The search in the dbt docs is pretty good, as is the project/database navigator on the left side on the site. We should only maintain these docs for things that are not in dbt, and be removing anything that is already in the dbt docs. Otherwise it's going to be a pain to maintain and they could easily get out of sync. I don't think it's worth the effort to keep them synced programmatically.

**Comment by charlie-costanzo** [comment]
@atvaccaro @lauriemerrell I made changes based on your feedback from before - take a look whenever you get a chance

**Review by lauriemerrell — COMMENTED** [review]
Hi Charlie -- thanks for the updates, I think that this looks a lot easier to maintain. One further step could be -- do we want to have a general "navigating the dbt docs" page that explains the database vs. project view mechanics, and each specific page could just link to that, and then for each data type it would just have the domain-specific stuff like the `Warehouse Schemas` section that's totally specific?

I'm just always a bit nervous when I see very similar text repeated on multiple pages, it makes me wonder about the maintenance logistics.

File: docs/datasets_and_tables/navigating_dbt_docs.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### The `Database` Perspective
+This allows you to view the dbt project as it exists in the warehouse.
+
+To examine the documentation from the `Database` perspective:
+
+1. Once at the [dbt docs homepage](https://dbt-docs.calitp.org/#!/overview), make sure that the `Database` tab is selected in the left-side panel
+1. In the same left-side panel, under the `Tables and Views` heading, click on `cal-itp-data-infra` which will expand
+1. Within that list, select the dataset schema of your choice
+1. From here, a dropdown list of tables will appear and you can select a table to view it's documentation
```
Comment (lauriemerrell) [inline docs/datasets_and_tables/navigating_dbt_docs.md:20]:
```suggestion
1. From here, a dropdown list of tables will appear and you can select a table to view its documentation
```

File: docs/datasets_and_tables/navigating_dbt_docs.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+### The `Project` Perspective
+This allows you to view the warehouse project as it exists as files in the repository.
+
+To examine the documentation for our tables from the `Project` perspective:
+
+* Once at the [dbt docs homepage](https://dbt-docs.calitp.org/#!/overview), make sure that the `Project` tab is selected in the left-side panel.
+    * To examine our `source` tables:
+        1. In the same left-side panel, find the `Sources` heading
+        1. From here, select the source that you would like to view
+        1. A dropdown list of tables will appear and you can select a table to view it's documentation
```
Comment (lauriemerrell) [inline docs/datasets_and_tables/navigating_dbt_docs.md:31]:
``` suggestion
        1. A dropdown list of tables will appear and you can select a table to view its documentation
```

File: docs/datasets_and_tables/navigating_dbt_docs.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    * To examine all of our other tables:
+        1. In the same left-side panel, under the `Projects`, heading click on `calitp_warehouse` which will expand.
+        1. Within that list, select `models`
+        1. From here, file directories will appear below.
+        1. Select the directory of your choice. A dropdown list of tables will appear and you can select a table to view it's documentation
```
Comment (lauriemerrell) [inline docs/datasets_and_tables/navigating_dbt_docs.md:37]:
```suggestion
        1. Select the directory of your choice. A dropdown list of tables will appear and you can select a table to view its documentation
```

File: docs/datasets_and_tables/views.md [hunk 4]
```diff
@@ -1,45 +1,33 @@
...
+
+## Database Schemas
+
+### Views
+| dataset name |
+| ------- |
+| `views.*` |
+| `mart_transit_database.*` |
+
+### Views Staging
```
Comment (lauriemerrell) [inline docs/datasets_and_tables/views.md:20]: I would actually remove this I think? Alternatively, would need to include Payments views staging, no?

File: docs/datasets_and_tables/gtfs_schedule.md [hunk 5]
```diff
@@ -1,15 +1,44 @@
...
 
-## DAGs Maintenance
+### Staging tables
+| Directory name |
+| ------- |
+| `gtfs_schedule_latest_only` |
 
-You can find further information on DAGs maintenance for GTFS Schedule data [on this page](gtfs-schedule-dags).
+:::{admonition} See Also: `views` documentation
+:class: tip
```
Comment (lauriemerrell) [inline docs/datasets_and_tables/gtfs_schedule.md:42]: I think the little `tip` style is so cute 🥺

File: docs/datasets_and_tables/navigating_dbt_docs.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+(navigating-dbt-docs)=
+# `Navigating the dbt Docs`
```
Comment (lauriemerrell) [inline docs/datasets_and_tables/navigating_dbt_docs.md:2]: Is there a reason to have this be `code style`? Wondering if the `s should be removed

File: docs/datasets_and_tables/gtfs_realtime.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+| `external_gtfs_rt` | Hive-partitioned external tables reading GTFS RT data and validation errors from GCS. |
+| `gtfs_rt_logs` | Data in the gtfs_rt_logs dataset in BigQuery, from logs sink. |
+| `gtfs_rt` | Data in the gtfs_rt dataset in BigQuery, generally produced by the rt_loader and rt_loader_files Airflow DAGs. |
+
+### Staging
+| dataset name | description |
+| ------- | ----------- |
+| `staging` | tables with a prefix `staging.stg_rt__*` |
+
+### Internal Tables
```
Comment (lauriemerrell) [inline docs/datasets_and_tables/gtfs_realtime.md:27]: Can we clarify what is meant by "internal" here? I'm not really sure what most of these are referring to... Wonder if this section can actually be deleted?

**Review by lauriemerrell — CHANGES_REQUESTED** [review]
Mostly a few small grammar / style questions, just one or two content things. Thanks again Charlie, sorry for all the back and forth here

File: docs/datasets_and_tables/gtfs_realtime.md (hunk 7 above)
Comment (charlie-costanzo) [reply docs/datasets_and_tables/gtfs_realtime.md:27]: @lauriemerrell that already existed, I just kept it. happy to delete...

File: docs/datasets_and_tables/navigating_dbt_docs.md (hunk 6 above)
Comment (charlie-costanzo) [reply docs/datasets_and_tables/navigating_dbt_docs.md:2]: @lauriemerrell yeah this was dum. it just made it stand out in the left nav as different from the rest of the sections but is unnecessary/misleading

File: docs/datasets_and_tables/gtfs_schedule.md (hunk 5 above)
Comment (charlie-costanzo) [reply docs/datasets_and_tables/gtfs_schedule.md:42]: I probably use it unnecessarily because I think it's so cute

**Comment by charlie-costanzo** [comment]
> Mostly a few small grammar / style questions, just one or two content things. Thanks again Charlie, sorry for all the back and forth here

no I appreciate it! would rather get it right

File: docs/datasets_and_tables/views.md (hunk 4 above)
Comment (charlie-costanzo) [reply docs/datasets_and_tables/views.md:20]: @lauriemerrell yeah I'm going to remove now

Comment (charlie-costanzo) [reply docs/datasets_and_tables/views.md]: @lauriemerrell I did add it to the `GTFS Schedule` doc though, as it looks like it's mostly just cleaned schedule data? let me know if I should remove

File: docs/datasets_and_tables/gtfs_schedule.md [hunk 8]
```diff
@@ -1,15 +1,46 @@
...
+| Directory name |
+| ------- |
+| `gtfs_schedule_history` |
+| `gtfs_type2` |
 
-## Dashboards
+### Staging tables
+| Directory name |
+| ------- |
+| `gtfs_schedule_latest_only` |
```
Comment (lauriemerrell) [inline docs/datasets_and_tables/gtfs_schedule.md:39]: I would actually move these under "views", sorry I didn't catch before (this is the data we publish on the open data portal)

File: docs/datasets_and_tables/views.md (hunk 4 above)
Comment (lauriemerrell) [reply docs/datasets_and_tables/views.md]: Yeah, that's ok

File: docs/datasets_and_tables/gtfs_realtime.md (hunk 7 above)
Comment (lauriemerrell) [reply docs/datasets_and_tables/gtfs_realtime.md]: Yeah,  I think deletion is cleaner ✅

**Review by lauriemerrell — CHANGES_REQUESTED** [review]
One more adjustment, sorry I missed it before. Otherwise I think this looks good.

File: docs/datasets_and_tables/gtfs_schedule.md (hunk 8 above)
Comment (charlie-costanzo) [reply docs/datasets_and_tables/gtfs_schedule.md]: no worries at all! let me know if my changes look right!

**Review by lauriemerrell — APPROVED** [review]
Looks good, thanks again Charlie!

**Comment by charlie-costanzo** [comment]
@atvaccaro whenever you're back it just looks like I still need your approval here to merge. absolutely no rush at all

**Review by atvaccaro — APPROVED** [review]

**Pull request merged**